scripts/Dockerfile.production

```diff
@@ -0,0 +1,44 @@
+FROM golang:1.11-stretch AS builder
+
+ARG BUILD=release
+
+WORKDIR /go/src/github.com/cozy/cozy-stack
+COPY . .
+RUN set -eux && \
+    go get -d -v ./... && \
+    bash -ex ./scripts/build.sh ${BUILD} && \
+    rm ./cozy-stack*.sha256
+
+
+
+FROM node:8-stretch-slim
+
+ENV COUCHDB_PROTOCOL=http \
+    COUCHDB_HOST=couchdb \
+    COUCHDB_PORT=5984 \
+    COUCHDB_USER=cozy \
+    COUCHDB_PASSWORD=cozy \
+    COZY_PASSWORD_RESET_INTERVAL=0
+
+
+COPY --from=builder /go/src/github.com/cozy/cozy-stack/cozy-stack* /usr/local/bin/cozy-stack
+COPY ./scripts/docker-entrypoint-production.sh /usr/local/bin/docker-entrypoint.sh
+COPY ./scripts/konnector-*-run.sh /usr/local/bin/
+COPY ./scripts/wait-for-it.sh /usr/local/bin/
+COPY ./scripts/cozy.yaml /etc/cozy/
```

Comment:
IIRC, it is recommended to minimize the number of `COPY` to avoid having an image with many layers

Reply:
Indeed. I'll squash line 26 & 27 but can't do better than that.